client/src/main/java/model/Ship.java

```diff
@@ -18,17 +17,15 @@
 
   private static final int START_SIZE = 1;
 
-  private final Mast[] masts;
+  private final List<Mast> masts;
   private final int shipLength;
   private int buildLength;
```

Comment:
Could it be replaced with masts.size() ?

Reply:
x